util/devel/test/vagrant/re2-supported.py

```diff
@@ -13,6 +13,7 @@
 # re2 requires c++11 for std atomics (and possibly other features). Use whether
 # std atomics are supported as a rough proxy for whether re2 should build
 compiler_val = chpl_compiler.get('target')
-std_atomics = compiler_utils.has_std_atomics(compiler_val)
+std_atomics = compiler_utils.has_std_atomics()
 
 sys.stdout.write('bundled' if std_atomics else 'none')
+sys.stdout.write('\n')
```

Comment:
I don't think we want a newline since this script is used to set `CHPL_RE2`. And we probably want to change `bundled` to `re2` now, since I think `bundled` was deprecated.

Reply:
It seems to work in the backtick context OK with the newline. I added the newline because on my system, Python wasn't outputting anything (presumably the stdout file is line buffered and closed before flushing the buffer?)

Reply:
Also I think you have re2 vs bundled backwards. Bundled is the current normal configuration default.

Reply:
> Also I think you have re2 vs bundled backwards. Bundled is the current normal configuration default.

Whoops, yeah you're right.